src/main/java/frc/robot/subsystems/drive/GyroIO.java

```diff
@@ -11,6 +11,12 @@ public static class GyroIOInputs {
     public double yawVelocityRadPerSec = 0.0;
     public double[] odometryYawTimestamps = new double[] {};
     public Rotation2d[] odometryYawPositions = new Rotation2d[] {};
+    public double VectorM;
+    public double pVectorM;
+    /*VectorA - Vector Angle, pVectorA - record previous value*/
+    public double VectorA;
+    public double pVectorA;
+    public double VectorDiff;
```

Comment:
these variable names are confusing and dont align with java conventions. Please rename to be clearer